collections-csharp/MergingDictionariesInCSharp/MergingDictionariesInCSharp.Benchmark/MergingMethodsBenchmark.cs

```diff
@@ -0,0 +1,93 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+using static MergingDictionariesInCSharp.Methods;
+
+[MemoryDiagnoser]
+[Orderer(SummaryOrderPolicy.FastestToSlowest)]
+[RankColumn]
+public class MergingMethodsBenchmark
+{
+    private Dictionary<int, string> _dictionaryA;
+    private Dictionary<int, string> _dictionaryB;
+
+    [GlobalSetup]
+    private void Setup()
+    {
+        _dictionaryA = new Dictionary<int, string>();
+        _dictionaryB = new Dictionary<int, string>();
+
+        Random random = new Random();
```

Comment:
Since we are not using a seed, this can be changed to use [Random.Shared](https://learn.microsoft.com/en-us/dotnet/api/system.random.shared?view=net-7.0). The same applies in the GenerateRandomValue method.

Reply:
This can also be a `var`.

Reply:
Changed to Random.Shared.
Used var.